### DISCUSSION TITLE
Elementary OS touchscreen compatible

### DISCUSSION THREAD

**Discussion by Ruben-Schadron** [elementary/triage#640] [Q&A]
It is possible to use Elementary OS on a 2-in-1 laptop tablet? Because now I'm trying it but some parts on the screen response sometimes.

**Comment by teamcons** [comment]
I did use 7 on a 2-in-1 tablet.
It was a bit awkward, but it did work, and was recognized out of the box.

**Reply by genseirin** [reply]
I'm also interested in that question.

@teamcons Have tried with version 8, is it still awkward? By "awkward" you mean the touchscreen does not work the way it should?

**Reply by teamcons** [reply]
> @teamcons Have tried with version 8, is it still awkward? By "awkward" you mean the touchscreen does not work the way it should?

No, it worked out of the box, 8 as well iirc
The awkwardness is mostly that the stylesheet, buttons and menus are built for a desktop computer - it can get annoying to snipe in buttons if you do not have a pen
i dont remember if the dock had gestures when on autohide - i think there is, but i did use it with always-there.
It was a bit strange with the dock, when you had the screen on horizontal, with all the space, and on vertical, where you couldnt display all icons at the same time. I recall i kind of limited pinned icons in the dock because of this - but then horizontal view had a very tiny dock..

overall, it's not that it does not work the way it should. It did work, and it was noticeable the team remembered touchscreens exist. But theres a couple uncut corners, maybe due to the team not using those (although i may be doing much more testing with touchscreens when/if we receive a Starlite), and also due to the UI not being as roundy and big as Gnome in places

**Reply by genseirin** [reply]
Thanks for the details!

**Reply by teamcons** [reply]
In retrospect i have to admit i may have forgotten back then that increasing font size also increases the UI...
It's not a bad experience. The thought of using eOS only using touch as pointer wouldnt worry me tbh.
